docs/source/WritingTests.rst

```diff
@@ -957,6 +957,9 @@ Avocado offers some options for the test writers to skip a test:
 Test ``skip()`` Method
 ----------------------
 
+.. warning:: `self.skip()` is under deprecation process. Please adjust
+   your tests to use the `self.cancel()` or the skip decorators instead.
```

Comment:
Also it'd be nice to add a date/version where we are going to deprecate the `skip`